Mobile-device-testing.md

```diff
@@ -22,6 +22,13 @@ A lot of developers have faced an issue "Request Host not whitelisted". To worka
 1. Once the simulator app icon appears in the dock, you can either pin it in the dock, or move/copy the app to a more convenient location for future access. This allows you to open the simulator without having to first open Xcode.
 1. Open Safari, then navigate to a locally-hosted developer instance to test Oppia on iOS.
 
+#### Directly run the simulator via command line
+
+1. See a list of available simulators using `xcrun simctl list`.
+Example output: `iPhone 11 (17DDCE3C-1V29-4251-BC19-21168CA1B259) (Shutdown)`
```

Comment:
Is it possible to place the command and output in (```) code blocks with 3 back ticks?
ex: https://github.com/oppia/oppia-web-developer-docs/blob/ed0ce8495760c25cde23333d446ede1d2b6354e8/Installing-Oppia-(Linux%3B-Python-3).md?plain=1#L52-L60

Reply:
That is for multiline code blocks. We don't need this since the code blocks here are single-lined.

Reply:
Since you're giving the cmd and an example output, do you think we can use a multiline code blocks?

Reply:
Done.